### PR TITLE
[EDMTP-388] Fixed workflow of dialog handlers

### PR DESCRIPTION
**Pull request by Alexey-Chernyakov**

### CODE REVIEW

File: aem-authoring-toolkit-plugin/src/test/com/exadel/aem/toolkit/test/custom/annotation/DialogAnnotationForOrderingTest.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface DialogAnnotationForOrderingTest {
+
```
Comment (smiakchilo) [inline aem-authoring-toolkit-plugin/src/test/com/exadel/aem/toolkit/test/custom/annotation/DialogAnnotationForOrderingTest.java:11]: (cosmetic) Please remove this redundant line

File: aem-authoring-toolkit-plugin/src/test/com/exadel/aem/toolkit/test/custom/handler/ordering/DialogManualOrderingTestCases.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+@SuppressWarnings("unused")
+// Expected order 2 -> 3 -> 1
+public class DialogManualOrderingTestCases {
+
+    private static final List<String> ATTRIBUTE_LIST = new ArrayList<>(3);
+
+    private static int counter = 0;
+
+    private static final String ORDER = "order";
```
Comment (smiakchilo) [inline aem-authoring-toolkit-plugin/src/test/com/exadel/aem/toolkit/test/custom/handler/ordering/DialogManualOrderingTestCases.java:33]: (cosmetic) Please avoid tautological constants' names. They are indeed senseless, and as a result people come and say we do not need constants at all). This is kinda `ATTRIBUTE_VALUE_ORDER`, etc.

File: aem-authoring-toolkit-plugin/src/test/com/exadel/aem/toolkit/test/custom/handler/ordering/DialogManualOrderingTestCases.java [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+import com.exadel.aem.toolkit.api.handlers.Target;
+import com.exadel.aem.toolkit.test.custom.annotation.DialogAnnotationForOrderingTest;
+
+@SuppressWarnings("unused")
+// Expected order 2 -> 3 -> 1
+public class DialogManualOrderingTestCases {
+
+    private static final List<String> ATTRIBUTE_LIST = new ArrayList<>(3);
+
+    private static int counter = 0;
```
Comment (smiakchilo) [inline aem-authoring-toolkit-plugin/src/test/com/exadel/aem/toolkit/test/custom/handler/ordering/DialogManualOrderingTestCases.java:31]: Please move this non-constant underneath the constants

**Review by smiakchilo — APPROVED** [review]

File: aem-authoring-toolkit-plugin/src/test/com/exadel/aem/toolkit/test/custom/annotation/DialogAnnotationForOrderingTest.java [hunk 4]
```diff
@@ -0,0 +1,12 @@
+package com.exadel.aem.toolkit.test.custom.annotation;
```
Comment (liubou-masiuk) [inline aem-authoring-toolkit-plugin/src/test/com/exadel/aem/toolkit/test/custom/annotation/DialogAnnotationForOrderingTest.java:1]: licence

**Review by liubou-masiuk — APPROVED** [review]

**Pull request merged**